modules/kubernetes/eks-core/variables.tf

```diff
@@ -340,3 +340,9 @@ variable "vpa_enabled" {
   type        = bool
   default     = true
 }
+
+variable "grafana_agent_enabled" {
```

Comment:
Should we do this or should we skip grafana_agent_enabled in eks since we haven't verified it yet and we probably need to do a few things. My thought was that I wanted to keep the networkpolicies as similar as possible as we talked about on yesterdays meeting.

Reply:
Keep it!